src/modules/legacy/components/Status/status.test.js

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import { mountWithRouterAndQueryClient } from 'src/utils/testHelpers';
 import TxBroadcaster from '@transaction/components/TxBroadcaster';
 import accounts from '@tests/constants/wallets';
+import { useCommandSchema } from '@network/hooks';
+import { mockCommandParametersSchemas } from 'src/modules/common/__fixtures__';
 import Status from './Status';
 
 jest.mock('@libs/wcm/hooks/useSession', () => ({
   respond: jest.fn(),
 }));
+jest.mock('@network/hooks');
```

Comment:
you should not ever mock entier directory 😭😭😭😭😭😭
its coused an issue all of the hooks on the directory will break on the test and in some how may effected on the other testes use jest.spyOn insted of mock

```js
import * as networkhook from @network/hooks

jest.spyOn(networkhook, 'useCommandSchema')

useCommandSchema.mockImplementation(() => ({
```

cc @eniolam1000752 @ikem-legend @reyraa @oskarleonard @ManuGowda